db/migrate/20210630201908_add_generated_date_to_dumps.rb

```diff
@@ -0,0 +1,5 @@
+class AddGeneratedDateToDumps < ActiveRecord::Migration[5.2]
+  def change
+    add_column :dumps, :generated_date, :datetime
+  end
+end
```

Comment:
Why we don't use the created_at?

Reply:
ok generated_date is the datetime the dump file is complete and exists in the db.

Reply:
created_at reflects the date that the file was downloaded, not the date the file itself was actually generated. This should give us a more accurate view of the age of the data itself.

Reply:
`generated_date` is taken from the filename, and reflects the date that the file was created by the SCSB system.